imagekit_test.go

```diff
@@ -262,45 +296,14 @@ func TestUrl(t *testing.T) {
 			if err != nil {
 				t.Errorf(err.Error())
 			}
-
-			if strings.Index(tc.url, "tr:") > -1 {
-				url, tr := extractTransformation(t, url)
-				expectedUrl, expectedTr := extractTransformation(t, tc.url)
-
-				if !urlsEquals(url, expectedUrl) {
-					t.Errorf("expected url: %s\ngot: %s", expectedUrl, url)
-				}
-
-				if !cmp.Equal(tr, expectedTr) {
-					t.Errorf("url: %s, expected tr: %s\ngot tr: %s", tc.url, expectedTr, tr)
-				}
+			if !urlsEquals(url, tc.url) {
+				t.Errorf("expected url: %s\ngot: %s", tc.url, url)
 			}
 		})
 	}
 
 }
 
-func extractTransformation(t *testing.T, url string) (urlResult string, trResult []string) {
```

Comment:
I removed this because it was not needed anymore. This function removes transformations from the path and checks whether they exist in the URL. Now, we are directly matching URLs.
This was needed earlier because the sequence of transformation was not fixed earlier refer [here](https://github.com/imagekit-developer/imagekit-go/pull/7/files#r1605897741)